pkg/virt-handler/cmd-client/client.go

```diff
@@ -193,20 +191,20 @@ func SocketMonitoringEnabled(socket string) bool {
 }
 
 func IsSocketUnresponsive(socket string) bool {
+	exists, _ := diskutils.FileExists(socket)
```

Comment:
Probably want to add a test to ` It("Detect unresponsive socket", func()`

Reply:
Added a unit test cases for a corner case that pod was evicted from node so the ghost record socket must be marked as unresponsive